docs/features.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Features
+---
+
+Below you can find a description of all the features that are available in Busola dashboard and their configuration examples:
+
+> **TIP:** The features in the list are ordered alphabetically.
+
+- **ADD_CLUSTER_DISABLED** – is used to enable or disable adding a cluster. If the function is enabled, you can only access your cluster in the BTP cockpit by adding it through the KUBECONFIG_ID feature. Link to the cockpit is configured under cockpitUrl.
+
+Default settings:
+
+```bash
+"ADD_CLUSTER_DISABLED":{
+  "isEnabled": false,
+  "config": {
+    "cockpitUrl":"https://account.staging.hanavlab.ondemand.com/cockpit"
+  }
+},
+```
+
+- **ADDONS** – is used to show or hide the Addons view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.
+  For the view to be shown, the feature must be enabled and all CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"ADDONS ":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"addons.kyma-project.io"
+    }
+  ]
+},
+```
+
+- **API_GATEWAY** – is used to show or hide the Api Gateway view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.
+  For the view to be shown, the feature must be enabled and all CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"API_GATEWAY ":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"gateway.kyma-project.io"
+    }
+  ]
+},
+```
+
+- **APPLICATIONS** – is used to show or hide the Applications view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.
+  For the view to be shown, the feature must be enabled and all CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"APPLICATIONS":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"applicationconnector.kyma-project.io"
+    }
+  ]
+},
+```
+
+- **BTP_CATALOG** – is used to show or hide the BTP Catalog view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.
+  For the view to be shown, the feature must be enabled, and all CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"BTP_CATALOG":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"services.cloud.sap.com"
+    }
+  ]
+},
+```
+
+- **CUSTOM_DOMAINS** – is responsible for displaying views of components such as DNS entries, DNS providers, gateways, issuers and certificates. Its functions can be visible only if all Custom Resource Defintions, defined in the selectors array, exist in a cluster.
```

Comment:
Shouldn't it be DNS Entries, DNS Providers, Gateways, Issuers and Certificates?

Reply:
I will change it.